src/components/Dropdown/index.spec.js

```diff
@@ -0,0 +1,94 @@
+import { describe, expect, it, beforeEach} from 'vitest';
+import { fireEvent, getByText } from '@testing-library/dom'
+import Dropdown from './index';
+
+const itemsMock = {
+  items: [
+    {
+      label: 'Raiva',
+      value: 'raiva',
+    },
+    {
+      label: 'Soninho',
+      value: 'soninho',
+    },
+    {
+      label: 'Castração',
+      value: 'castracao',
+    },
+  ]
+}
+;
+
+describe('Dropdown', () => {
+  let dropdown;
+  let clickEvent;
+
+  beforeEach(() => {
+    dropdown = new Dropdown(itemsMock);
+    clickEvent = new Event('click');
+  });
+  it('is a Function', () => {
+    expect(Dropdown).toBeInstanceOf(Function);
+  });
+
+  it('returns an object', () => {
+    expect(new Dropdown(itemsMock)).toBeInstanceOf(Object);
+  });
+
+  it('the component must contain additional items',() => {
+    expect(dropdown.selected.get('dropdown-options').children.length).toBe(3);
+  });
+
+  it('should open the container when clicked', () => {
```

Comment:
Não usar `should` a ideia da função `it` é justamente pra não precisarmos ficar repetindo o `should`. No caso, como mencionado antes, usar o imperativo.

`it('opens the container when clicked', () => ...)`